### PR TITLE
[RedSuns Review] Maintenance Tutorials(MCTWrapper)

### PR DESCRIPTION
**Pull request by ueno-hiroshi002**
## Pull Request Description:
* Minimized parameters for a simple demo.  
* Refactored text and source code.
* Set versions at installing below libraries, same of #1632
    * tensorflow==2.15
    * onnx==1.17.0
* Add how to downgrade python3.12 to python3.11 for tensorflow in Colab environment, same of #1632

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

**Review by KazunoriSumiya — APPROVED** [review]

**Review by kawakami-masaki0 — APPROVED** [review]

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb [hunk 1]
```diff
@@ -5,22 +5,34 @@
...
-    "2. **Dataset Preparation**: Load and prepare ImageNet validation dataset with representative data generation\n",
-    "3. **PTQ Implementation**: Execute basic Post-Training Quantization with 8-bit precision and bias correction\n",
-    "4. **PTQ + Mixed Precision**: Apply intelligent bit-width allocation based on layer sensitivity analysis (75% compression ratio)\n",
-    "5. **GPTQ Implementation**: Perform gradient-based optimization with 5-epoch fine-tuning for enhanced accuracy\n",
-    "6. **GPTQ + Mixed Precision**: Combine gradient optimization with mixed precision for optimal accuracy-compression trade-off\n",
-    "7. **Performance Evaluation**: Comprehensive accuracy assessment and comparison across all quantization methods\n",
-    "8. **Results Analysis**: Compare model sizes, inference accuracy, and quantization trade-offs\n",
+    "- **Setup**: Import required libraries and configure MCT with MobileNetV2 model\n",
+    "- **Dataset Preparation**: Load and prepare ImageNet validation dataset with representative data generation\n",
+    "- **Model Quantization using MCTWrapper**: Quantize the float model using MCTWrapper with four methods.\n",
```
Comment (yt0705) [inline tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: > MCTWrapper with four methods.

Delete period

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb [hunk 2]
```diff
@@ -378,15 +328,14 @@
...
+    "    # Parameter configuration\n",
     "    param_items = [\n",
-    "        ['sdsp_version', '3.14'],  # The version of the SDSP converter.\n",
-    "        ['num_of_images', 5],  # Number of epochs for gradient-based fine-tuning.\n",
-    "        ['use_hessian_based_scores', False],  # Use Hessian-based sensitivity scores for layer importance.\n",
-    "        ['weights_compression_ratio', 0.75],  # Target compression ratio for model weights (75% of original size.\n",
-    "        ['save_model_path', './qmodel_PTQ_Keras_mixed_precision.keras']  # Path to save the quantized model.\n",
+    "        ['sdsp_version', '3.14'],                                         # Version of the SDSP converter\n",
+    "        ['num_of_images', 5],                                             # Number of images for calibration\n",
+    "        ['weights_compression_ratio', 0.5],                               # Compression ratio for weights\n",
```
Comment (yt0705) [inline tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: > Compression ratio for weights

Please provide a detailed explanation of what the purpose of the parameter is.

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb [hunk 3]
```diff
@@ -507,18 +437,15 @@
...
-    "        ['n_epochs', 5],  # Number of epochs for gradient-based fine-tuning.\n",
-    "        ['optimizer', None],  # Optimizer to use during fine-tuning.\n",
-    "        ['num_of_images', 5],  # Number of images to use for calibration.\n",
-    "        ['use_hessian_based_scores', False],  # Whether to use Hessian-based scores for layer importance.\n",
-    "        ['weights_compression_ratio', 0.75],  # Compression ratio for weights.\n",
-    "        ['save_model_path', './qmodel_GPTQ_Keras_mixed_precision.keras']  # Path to save the quantized model.\n",
+    "        ['sdsp_version', '3.14'],                                          # Version of the SDSP converter\n",
+    "        ['n_epochs', 5],                                                   # Number of epochs for GPTQ optimization\n",
+    "        ['num_of_images', 5],                                              # Number of images for calibration\n",
+    "        ['weights_compression_ratio', 0.5],                                # Compression ratio for weights\n",
```
Comment (yt0705) [inline tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: > Compression ratio for weights

Please provide a detailed explanation of what the purpose of the parameter is.

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb [hunk 4]
```diff
@@ -692,11 +592,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Evaluate GPTQ quantized model accuracy\n",
-    "print(\"\\n=== GPTQ Model Evaluation ===\")\n",
+    "# GPTQ quantized model\n",
```
Comment (yt0705) [inline tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: > GPTQ quantized

The term “quantization” is used twice consecutively.

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb [hunk 5]
```diff
@@ -664,11 +566,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Evaluate PTQ quantized model accuracy\n",
-    "print(\"\\n=== PTQ Model Evaluation ===\")\n",
+    "# PTQ quantized model\n",
```
Comment (yt0705) [inline tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: > PTQ quantized

The term “quantization” is used twice consecutively.

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb [hunk 6]
```diff
@@ -313,17 +275,12 @@
...
     "    # Parameter configuration for PTQ\n",
     "    param_items = [\n",
-    "        ['sdsp_version', '3.14'],  # The version of the SDSP converter.\n",
-    "        ['activation_error_method', QuantizationErrorMethod.MSE],  # Error metric for activation.\n",
-    "        ['weights_bias_correction', True],  # Enable bias correction for weights.\n",
-    "        ['z_threshold', float('inf')],  # Threshold for zero-point quantization.\n",
-    "        ['linear_collapsing', True],  # Enable linear layer collapsing optimization.\n",
-    "        ['residual_collapsing', True],  # Enable residual connection collapsing.\n",
-    "        ['save_model_path', './qmodel_PTQ_Keras.keras']  # Path to save the quantized model.\n",
+    "        ['sdsp_version', '3.14'],                          # Version of the SDSP converter\n",
```
Comment (yt0705) [inline tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: > Version of the SDSP converter

Please provide a detailed explanation of what the purpose of the parameter is.

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb [hunk 7]
```diff
@@ -441,15 +383,13 @@
...
     "\n",
-    "    # Parameter configuration for GPTQ\n",
+    "    # Parameter configuration\n",
     "    param_items = [\n",
-    "        # Platform configuration\n",
-    "        ['sdsp_version', '3.14'],  # The version of the SDSP converter.\n",
-    "        ['n_epochs', 5],  # Number of epochs for gradient-based fine-tuning.\n",
-    "        ['optimizer', None],  # Optimizer to use during fine-tuning.\n",
-    "        ['save_model_path', './qmodel_GPTQ_Keras.keras']  # Path to save the quantized model.\n",
+    "        ['sdsp_version', '3.14'],                          # Version of the SDSP converter\n",
```
Comment (yt0705) [inline tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: > Version of the SDSP converter

Please provide a detailed explanation of what the purpose of the parameter is.

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb (hunk 5 above)
Comment (ueno-hiroshi002) [reply tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: fixed at 31cfe06

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb (hunk 4 above)
Comment (ueno-hiroshi002) [reply tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: fixed at 31cfe06

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb (hunk 3 above)
Comment (ueno-hiroshi002) [reply tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: fixed at 31cfe06

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb (hunk 2 above)
Comment (ueno-hiroshi002) [reply tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: fixed at 31cfe06

File: tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb (hunk 1 above)
Comment (ueno-hiroshi002) [reply tutorials/notebooks/mct_features_notebooks/keras/example_keras_mct_wrapper.ipynb]: fixed at 31cfe06
